tools/setup-startup-profiling.py

```diff
@@ -56,7 +56,8 @@ def push(id, filename):
         run(['adb', 'shell', 'am', 'set-debug-app', '--persistent', id])
         print('\nStartup profiling enabled on all future start ups, possibly even after reinstall.')
         print('Call script with `deactivate` to disable it.')
-        print('DON\'T FORGET TO ENABLE \'Remote debugging via USB\' IN THE APP SETTINGS!')
+        print('DON\'T FORGET TO DISABLE \'Remote debugging via USB\' IN THE APP SETTINGS!')
```

Comment:
Two thoughts:
1. it might be clearer if we say exactly when it needs to happen: e.g. "DISABLE 'Remote debugging via USB' BEFORE STARTING THE APP & RE-ENABLE TO CAPTURE THE PROFILE"
2. it'd be more memorable if we explained why that's important: e.g. "There is additional overhead if 'Remote debugging via USB' is enabled during start up."

Thanks for putting this together! 😄

Reply:
TY! Updated it, let me know if it needs additional rephrasing.